packages/canvas/canvas-renderer/src/CanvasRenderer.ts

```diff
@@ -152,6 +152,16 @@ export class CanvasRenderer extends AbstractRenderer
         this.resize(this.options.width, this.options.height);
     }
 
+    /**
+     * Adds a new system to the renderer. It does nothing in the CanvasRenderer.
+     *
+     * @extends PIXI.AbstractRenderer#addSystem
```

Comment:
Also, @extends isn't useful here either. Can be removed.

Reply:
Removed.